ghost/core/test/e2e-api/members/donation-checkout-session.test.js

```diff
@@ -329,4 +329,39 @@ describe('Create Stripe Checkout Session for Donations', function () {
 
         latestStripePrice.get('nickname').should.have.length(250);
     });
+    it('Can create a checkout session with a personal note included', async function () {
```

Comment:
This test is worthless.  It's always going to pass because the body is just the returned url.  I need a clue, please. :)

Reply:
Hey @cathysarisky! 

I think you're right about testing the different parsing scenarios at the unit level, here I think the only thing we'd like to test is that we can pass in a `personalNote` param and that doesn't break the checkout behaviour, end-to-end. 

Curious why we're passing the personal note under `custom_fields` in the test? Aren't we passing it as a top-level `personalNote` in the code?

Reply:
Good catch, thank you.  I think I got confused between content of that field and the label. *blush*